src/main/webapp/forms/formUtilities.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Sets one form input's value to the given value.
+ */
+function setFormInputValue(input, value) {
```

Comment:
Nit: maybe rename to `setFormInputValueOrEmpty`?

Reply:
Done.